examples/ltc_backend/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+###########################################################################
+# Setup PyTorch
+###########################################################################
+
+list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/python/torch_mlir/cmake/modules")
+include(TorchMLIRPyTorch)
+TorchMLIRProbeForPyTorchInstall()
```

Comment:
I think this recently changed with https://github.com/llvm/torch-mlir/pull/955

Reply:
yes please use TorchMLIRConfigureLibTorch(). We would like to remove the need to probe TorchMLIRProbeForPyTorchInstall() and also remove the need for PyTorch during our build.